### PR TITLE
Add new known issues and workarounds/fixes

### PR DESCRIPTION
**Pull request by MoteHue**

### CODE REVIEW

File: doc/source/operations/upgrading.rst [hunk 1]
```diff
@@ -107,6 +115,20 @@ Known issues
 * The OVN sync repair tool removes metadata ports, breaking OVN load balancers.
   See `LP#2038091 <https://bugs.launchpad.net/neutron/+bug/2038091>`__.
 
+* When you try to generate config before the 2023.1 upgrade (i.e. using 2023.1
+  Kolla-Ansible but still running Zed kolla-toolbox), it will fail on Octavia.
+  This patch is needed to fix this:
+  https://review.opendev.org/c/openstack/kolla-ansible/+/905500
+
+* If you run ``kayobe overcloud service upgrade`` twice, it will cause shard
```
Comment (Alex-Welsh) [inline doc/source/operations/upgrading.rst:123]: Would be nice to also have an easy command to work out whether this is an issue for a given deployment

File: doc/source/operations/upgrading.rst [hunk 2]
```diff
@@ -68,6 +68,14 @@ Some things to watch out for:
   will not be granted those roles. This may include the ``reader`` role, which
   is referenced in some of the new secure RBAC policies.  See `Keystone bug
   2030061 <https://bugs.launchpad.net/keystone/+bug/2030061>`_.
+* If you have overwritten ``[auth] tempest_roles`` in your Tempest config, such
+  as to add the ``creator`` role for Barbican, you will need to also add the
+  ``member role``. eg:
+
+  .. code-block:: console
```
Comment (Alex-Welsh) [inline doc/source/operations/upgrading.rst]: Not really console. Not sure if `ini` is an option. Maybe just omit?

**Review by Alex-Welsh — CHANGES_REQUESTED** [review]

File: doc/source/operations/upgrading.rst (hunk 1 above)
Comment (MoteHue) [reply doc/source/operations/upgrading.rst:123]: Good call, done

File: doc/source/operations/upgrading.rst (hunk 2 above)
Comment (MoteHue) [reply doc/source/operations/upgrading.rst]: Done

**Review by Alex-Welsh — APPROVED** [review]

**Review by markgoddard — APPROVED** [review]

**Pull request merged**